test/Hangman.test.js

```diff
@@ -85,79 +85,73 @@ contract('Hangman', async (accounts) => {
               assert.equal(input.toNumber(), 2, "expected value is incorrect");
           });
 
-          it("Test make chracter guess example 2", async () => {
+          it("Test makeCharGuess example 2", async () => {
               await truffleAssert.passes(
                 hangmanContract.makeCharGuess(ethers.utils.toUtf8Bytes("l")),
-                "Transaction failed");
+                "Transaction makeCharGuess failed");
               var input = await hangmanContract.playerInput.call();
               assert.equal(input.toNumber(), 12, "expected value is incorrect");
           });
 
-          it("Test make chracter guess example 3", async () => {
+          it("Test makeCharGuess example 3", async () => {
               await truffleAssert.passes(
                 hangmanContract.makeCharGuess(ethers.utils.toUtf8Bytes("h")),
-                "Transaction failed");
+                "Transaction makeCharGuess failed");
               var input = await hangmanContract.playerInput.call();
               assert.equal(input.toNumber(), 1, "expected value is incorrect");
           });
 
-          it("Test make chracter guess example o", async () => {
+          it("Test makeCharGuess example o", async () => {
               await truffleAssert.passes(
                 hangmanContract.makeCharGuess(ethers.utils.toUtf8Bytes("o")),
-                "Transaction failed");
+                "Transaction makeCharGuess failed");
               var input = await hangmanContract.playerInput.call();
               assert.equal(input.toNumber(), 16, "expected value is incorrect");
           });
       });
 
-      describe("Test make character guess", async () => {
-          it("Test make character guess with unknown char", async () => {
-              await truffleAssert.passes(
-                hangmanContract.makeCharGuess(ethers.utils.toUtf8Bytes("e")),
-                "Transaction failed");
-              var input = await hangmanContract.playerInput.call();
-              assert.equal(input.toNumber(), 2, "expected value is incorrect");
+      describe("Test makeCharGuess", async () => {
+          it("Test makeCharGuess with unknown char", async () => {
+            let tx = await hangmanContract.makeCharGuess(ethers.utils.toUtf8Bytes("e"))
+            await truffleAssert.eventEmitted(tx, 'TurnTaken', {isValidInput: true});
+
+            var input = await hangmanContract.playerInput.call();
+            assert.equal(input.toNumber(), 2, "expected value is incorrect");
           });
 
-          it("Test make character guess with known char", async () => {
-              await truffleAssert.passes(
-                hangmanContract.makeCharGuess(ethers.utils.toUtf8Bytes("e")),
-                "Transaction failed");
-              await truffleAssert.reverts(
-                hangmanContract.makeCharGuess(ethers.utils.toUtf8Bytes("e")),
-                "character has aleady been guessed");
```

Comment:
this logic should be retained. When the same character is called it should be rejected.